hack/verify-api-groups.sh

```diff
@@ -94,6 +94,7 @@ done
 packages_without_install=(
 	"k8s.io/kubernetes/pkg/apis/abac"
 	"k8s.io/kubernetes/pkg/apis/admission"
+	"k8s.io/kubernetes/pkg/apis/componentconfig" # TODO: Remove this package completely and from this list
```

Comment:
under which conditions?

Reply:
when the directory is removed we can get rid of this. Is the comment clear enough?

Reply:
missed the "and".  it's ok.